cmd/marker/main.go

```diff
@@ -20,25 +20,39 @@ import (
 	"time"
 
 	"github.com/golang/glog"
+	"github.com/kubevirt/bridge-marker/pkg/cache"
 	"github.com/kubevirt/bridge-marker/pkg/marker"
+	"k8s.io/apimachinery/pkg/util/wait"
 )
 
 func main() {
 	nodeName := flag.String("node-name", "", "name of kubernetes node")
-	const defaultPollInterval = 60
-	pollInterval := flag.Int("update-interval", defaultPollInterval, fmt.Sprintf("interval between updates in seconds, %d by default", defaultPollInterval))
+	const defaultUpdateInterval = 60
+	updateInterval := flag.Int("update-interval", defaultUpdateInterval, fmt.Sprintf("interval between updates in seconds, %d by default", defaultUpdateInterval))
+	const defaultReconcileInterval = 10
+	reconcileInterval := flag.Int("reconcile-interval", defaultReconcileInterval, fmt.Sprintf("interval between node bridges reconcile in minutes, %d by default", defaultReconcileInterval))
```

Comment:
ditto, let's use duration for for intervals from the beggining so we can express quantity and unit at the same variable.

Reply:
Done